lua/lazyvim/plugins/extras/lang/toml.lua

```diff
@@ -0,0 +1,31 @@
+recommended = function()
```

Comment:
This should be inside the returned table below :)

Reply:
oopsie.. I hope I got it right this time